### ISSUE TITLE
Use other webproviders outside of autoscout

### ISSUE THREAD

**Issue by danielkagemann** [Elemetric/cadavis#13]
_From @Practical-DQ on January 31, 2016 8:46_

_Copied from original issue: Elemetric/as24#31_

**Comment by danielkagemann** [comment]
_From @Practical-DQ on January 31, 2016 8:48_

There are othr car web providers, like .... Mobile.de, autotrader.co.uk how much adaptation work is needed to onboard the other services

**Comment by danielkagemann** [comment]
_From @Practical-DQ on February 7, 2016 10:6_

Extend other providers to include Schwacke or wirkaufendeinauto.de

**Comment by Practical-DQ** [comment]
yes, ideally because the more the data is mixed, the rounder the picture becomes
